name: test read python error messages

on:
  schedule:
    # Runs "at minute 55 past every hour" (see https://crontab.guru)
    - cron: "00 15 * * *"
  workflow_dispatch:

permissions:
  contents: read

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - name: Set up Python
        uses: actions/setup-python@v5
        with:
          python-version: "3.13"
      - name: Run 
        run: |
          python main.py
      - name: Print error message
        if: ${{ failure() }}
        run: |
          echo "Error message: ${{ steps.Run.outputs.error_message }}"